test/darwin.js

```diff
@@ -482,7 +482,7 @@ if (!(process.env.CI && process.platform === 'win32')) {
     t.deepEqual(plistObj.ElectronAsarIntegrity, {
       'Resources/app.asar': {
         algorithm: 'SHA256',
-        hash: '27f2dba4273f6c119000ec7059c27d86e27306d5dbbb83cfdfb862d92c679574'
+        hash: '2ef4c03f6fc8adf03f782b40fddcfcdd26d1f3f3669ecedffeea5de5d28897c3'
```

Comment:
I'm oretty sure that this hash changed because I updated the `basic` app that `baseOpts` is passing into the test. Tested this locally on an x64 machine, and the hash is consistently returning this value both locally and in CI. However, if this doesn't seem to pass the sniff test, let me know, happy to take another look.

Reply:
> I'm oretty sure that this hash changed because I updated the basic app that baseOpts is passing into the test.

Might want to add a comment to this effect for future folks (and also how to find out the new value).